README.md

```diff
@@ -22,6 +22,12 @@
 - Implement submit button
 - Use async/await features to consume the API
 
+### Milestone 3:
+
+- Improve look and feel
+- Add styling
+- Maintain general layout of wireframe
+
 ## Built With
 - HTML, SCSS, Bootstrap, Webpack, Javascript  
 
```

Comment:
- [ ] It would be great if you add all that's needed for running the app with the API(s) it consumes. It should have instructions about where and how to generate any necessary keys, credentials, how to set, store, and generate them(if necessary) on the README file. It's an easy requirement but important to have a more professional project. 🐶